src/ucp/wireup/wireup.c

```diff
@@ -683,6 +673,7 @@ ucp_wireup_process_reply(ucp_worker_h worker, ucp_ep_h ep,
          */
         status = ucp_wireup_connect_local(ep, remote_address, NULL);
         if (status != UCS_OK) {
+            ucp_ep_set_failed_schedule(ep, UCP_NULL_LANE, status);
```

Comment:
this is the main fix right?

Reply:
yes, this is the main fix and above in the function `ucp_wireup_process_request()`